_posts/2022-12-05-solidity-core-team-updates.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+published: true
+title: 'Solidity Core Team Updates'
+date: '2022-12-05'
+author: Solidity Team
+category: Announcements
+---
+
+More than two years have passed since we [introduced Solidity core team members](https://blog.soliditylang.org/2020/09/18/meet-the-team/) on the blog and we realized it is high time for some updates: Meet new team members, find out who moved on to other adventures and learn about recent changes in the team structure!
+
+Before we dive in, a reminder that the Solidity programming language and compiler are **open-source community projects**. This post dives into the core team that leads the development. Nevertheless, we cannot stress enough how invaluable community contributions and interactions are for Solidity. We value feedback and input and want to thank our community for the continued support!
+
+## Thank you to our alumni 💐
+
+Like in any other project, there naturally comes a time for some team members to move on to other challenges. Since our last update, we have bid farewell to Christian (@christianparpart), Damian (@wechman), Đorđe (@mijovic), Hari (@hrkrshnn), Mathias (@marenz),  and Nishant (@nishant-sachdeva). We were extremely glad to have them and wish them all the best for their future! Luckily, some alumni remain in our orbit and continue to contribute with code and input to language design discussions.
+
+## Introducing new team members 💝
+
+On a happy note, we are excited to introduce new core team members who have joined us along the way. Please meet Matheus, Nikola, Nuno and Rodrigo!
+
+In order to give you a chance to get to know them better, we asked all new joiners to respond to a selection of questions. Read their responses below. 
+
+### Matheus - @[matheusaaguiar](https://github.com/matheusaaguiar)
+
+#### What is your professional background?
+
+_I have a BSc in Computer Science and a MSc in Artificial Intelligence. After finishing college, I worked as a C++ backend developer on a small educational game project for about 2 years. Subsequently, I worked as a Data Structure and Algorithms teacher for 8 years. I joined the Solidity team in May 2022 and am working on the Solidity Compiler since._
```

Comment:
```suggestion
_I have a BSc in Computer Science and an MSc in Artificial Intelligence. After finishing college, I worked as a C++ backend developer on a small educational game project for about 2 years. Subsequently, I worked as a Data Structure and Algorithms teacher for 8 years. I joined the Solidity team in May 2022 and have been working on the Solidity Compiler since._
```